test-network-function/certification/suite.go

```diff
@@ -173,3 +174,44 @@ func testOperatorCertificationStatus() {
 		}
 	})
 }
+
+func testCSICertified(env *configpkg.TestEnvironment) {
+	testID := identifiers.XformToGinkgoItIdentifier(identifiers.TestCSIOperatorIsCertifiedIdentifier)
+	ginkgo.It(testID, func() {
```

Comment:
Now that https://github.com/test-network-function/test-network-function/commit/ae87316f0ef5c33f55219927caaad625fa675aa1 was merged, could you change this line to be:

    ginkgo.It(testID, ginkgo.Label(testID), func() {

? That will give the test a label, so it can be used to filter.